### PR TITLE
New IT branch

### PR DESCRIPTION
**Pull request by shermandickman**
New IT Branch

### CODE REVIEW

File: it/mail/chrome/communicator/wallet/wallet.properties [hunk 1]
```diff
@@ -61,8 +61,8 @@ SaveTheseValuesEncrypted = Utilizza il Gestore password per ricordare questi dat
 SaveThisValueObscured = Utilizza il Gestore password per ricordare questo valore.
 SaveThisValueEncrypted = Utilizza il Gestore password per ricordare questo valore.
 SaveThisPasswordObscured= Utilizza il Gestore password per ricordare questa password.
-SaveThisPasswordEncrypted = Utilizza Gestione password per ricordare questa password.
-xSaveTheseValuesObscured = Ricorda questi dati. I dati archiviati non sono protetti da password.
+SaveThisPasswordEncrypted = Utilizzare Gestione password per ricordare questa password.
```
Comment (fenderman79) [inline it/mail/chrome/communicator/wallet/wallet.properties]: CORRECTIONS NEEDED. 

In order to improve the consistency of the text, please use "Utilizza" instead of "Utilizzare"

File: it/mail/chrome/communicator/wallet/wallet.properties [hunk 2]
```diff
@@ -61,8 +61,8 @@ SaveTheseValuesEncrypted = Utilizza il Gestore password per ricordare questi dat
...
+xSaveTheseValuesObscured = Ricorda questi dati. I dati archivati non sono protetti da password.
```
Comment (fenderman79) [inline it/mail/chrome/communicator/wallet/wallet.properties]: CORRECTIONS NEEDED. 

There is a typo: please use "archiviati" instead of "archivati".

File: it/mail/chrome/communicator/wallet/wallet.properties [hunk 3]
```diff
@@ -79,5 +79,5 @@ NotConverted = Impossibile convertire i dati archiviati.
...
+Caveat = Salvare password e altri dati sensibili## Il Gestore password e il Gestore moduli salveranno le password, i nomi utenti e altri dati sensibili e li inseriranno per te automaticamente quando sono richiesti.#Questi dati riservati sono archiviati sul tuo computer in un file che è difficile, ma non impossibile, da leggere.#Se altri utenti hanno accesso al tuo computer, puoi proteggere con password i dati sensibili archiviati scegliendo una Password principale.#Se scegli di proteggere con password i tuoi dati riservati, ti sarà richiesto di fornire di volta in volta la tua Password principale. Questo metodo fornisce una miglior sicurezza ma è leggermente più scomodo.
```
Comment (fenderman79) [inline it/mail/chrome/communicator/wallet/wallet.properties]: CORRECTIONS NEEDED. 

Wrong plural: use "nomi utente" instead of "nomi utenti".

File: it/mail/chrome/messenger/AccountWizard.dtd [hunk 4]
```diff
@@ -1,61 +1,61 @@
...
 <!ENTITY accountTypeMail.accesskey "c">
-<!ENTITY accountTypeNews.label "Account gruppi di discussione">
+<!ENTITY accountTypeNews.label "Account Newsgroup">
 <!ENTITY accountTypeNews.accesskey "o">
 
 <!-- Entities for Identity page -->
 
 <!ENTITY identityTitle.label "Identità">
-<!ENTITY identityDesc.label "Ogni account ha una propria identità, cioè l'insieme di informazioni che permettono al destinatario di identificare il mittente.">
+<!ENTITY identityDesc.label "Ogni account ha un'identità, che consiste nelle informazioni che permettono agli alri di identificarti quando ricevono i tuoi messaggi.">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/AccountWizard.dtd]: CORRECTIONS NEEDED. 

There is a typo: please replace "alri" with "altri".

File: it/mail/chrome/messenger/AccountWizard.dtd [hunk 5]
```diff
@@ -66,57 +66,57 @@
...
+<!ENTITY completionTitle.label "Congratulazioni!">
+<!ENTITY completionText.label "Verifica la correttezza delle informazioni sottostanti.">
+<!ENTITY serverTypePrefix.label "Tipo server in entrata:">
+<!ENTITY serverNamePrefix.label "Nome server in entrata:">
 <!ENTITY smtpServerNamePrefix.label "Nome server in uscita (SMTP):">
 <!ENTITY newsServerNamePrefix.label "Nome server delle news (NNTP):">
-<!ENTITY downloadOnLogin.label "Scarica subito i messaggi">
-<!ENTITY deferStorageDesc.label "Togliere la spunta per conservare la posta di questo account in una cartella separata. Questo sposterà l'account al primo livello. In caso contrario farà parte della cartella 'Posta in arrivo' globale delle Cartelle locali.">
+<!ENTITY downloadOnLogin.label "Scarica i messaggi ora">
+<!ENTITY deferStorageDesc.label "Togli la spunta per archiviare le e-mail di questo account inella sua cartella. Questo farà apparire questo account comeun account di livello superiore. Altrimenti, farà parte dell'account in entrata globale delle cartelle locali.">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/AccountWizard.dtd]: CORRECTIONS NEEDED. 

There is a typo: please replace "inella" with "nella".

File: it/mail/chrome/messenger/FilterEditor.dtd [hunk 6]
```diff
@@ -39,49 +39,49 @@
...
 -->
 
-<!ENTITY playSound.label "Play Sound">
-<!ENTITY notPendingStatus.label "Not Pending">
-<!ENTITY noSoundSelected.label "Select a Sound">
-<!ENTITY noScriptsInstalled.label "No Scripts installed">
+<!ENTITY playSound.label "Riproduci suono">
+<!ENTITY notPendingStatus.label "Non in attesa">
+<!ENTITY noSoundSelected.label "Seleziona un suono">
+<!ENTITY noScriptsInstalled.label "Nessun script installato">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/FilterEditor.dtd]: CORRECTIONS NEEDED. 

There is a typo. Please replace "Nessun" with "Nessuno".

File: it/mail/chrome/messenger/FilterEditor.dtd [hunk 7]
```diff
@@ -39,49 +39,49 @@
...
+<!ENTITY removeTag.label "Rimuovi argomento">
+<!ENTITY reminderStatus.label "Promemoria">
+<!ENTITY markMessagePending.label "Contrassegna come in attesa">
+<!ENTITY noScriptSelected.label "Seleziona un AppleScript">
+<!ENTITY globalFilterChekbox.label "Questo è un filtro globale">
+<!ENTITY readStatus.label "Letto">
+<!ENTITY launchScript.label "Esegui AppleScript">
+<!ENTITY openInFinder.label "Apri nel Finder">
+<!ENTITY pendingStatus.label "In attesa">
+<!ENTITY notReminderStatus.label "Not promemoria">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/FilterEditor.dtd]: CORRECTIONS NEEDED.

There is a typo. Please replace "Not" with "Non".

File: it/mail/chrome/messenger/accountCreation.dtd [hunk 8]
```diff
@@ -60,13 +60,13 @@
 <!ENTITY changeSettings.label            "Cambia impostazioni">
 <!ENTITY changeSettings.accesskey        "m">
 
-<!ENTITY pb.contactYourProvider.description "&brandShortName; può consentire di collegarti alla tua e-mail utilizzando le configurazioni fornite. Tuttavia, dovresti contattare l'amministratore o il fornitore di servizi e-mail in merito a queste connessioni non corrette.">
+<!ENTITY pb.contactYourProvider.description "&brandShortName; può consentirti di collegarti alla tua e-mail utilizzando le configurazioni fornite. Tuttavia, dovresti contattare il tuo amministratore di sistema o il tuo fornitore di servizi e-mail in merito a queste connessioni non corrette.">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/accountCreation.dtd]: CORRECTIONS NEEDED. 

Too many personal pronouns. Please replace "consentirti" with "consentire".

File: it/mail/chrome/messenger/accountCreation.dtd [hunk 9]
```diff
@@ -60,13 +60,13 @@
...
 
-<!ENTITY insecureUnencrypted.description "La tua e-mail e le informazioni di autenticazione sono inviate senza crittografia, pertanto sia la password che il messaggio potrebbero essere facilmente letti da altri utenti. &brandShortName; consentirà di collegarti alla tua e-mail, ma dovresti contattare il fornitore di servizi e-mail per configurare il server con un certificato affidabile.">
-<!ENTITY insecureSelfSigned.description "Il server utilizza un certificato non affidabile, pertanto non possiamo essere sicuri che qualcuno non stia intercettando il traffico tra &brandShortName; e il server. &brandShortName; consentirà di collegarti alla tua e-mail, ma dovresti assolutamente richiedere al fornitore di servizi e-mail di configurare il server con un certificato affidabile.">
+<!ENTITY insecureUnencrypted.description "La tua e-mail e le tue informazioni di autenticazione sono inviate senza crittografia, pertanto la tua password e il tuo messaggio potrebbero essere facilmente letti da altri utenti. &brandShortName; ti consentirà di collegarti alla tua e-mail ma dovresti contattare il tuo fornitore di servizi e-mail per configurare il server con un certificato affidabile.">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/accountCreation.dtd]: CORRECTIONS NEEDED. 

Too many personal pronouns. Please remove "tue" before "informazioni di autenticazione".

File: it/mail/chrome/messenger/accountCreation.properties [hunk 10]
```diff
@@ -41,7 +41,7 @@ config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
 config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
 incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
 outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
-please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.
+please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/accountCreation.properties]: CORRECTIONS NEEDED. 

Elision and apocopation should be avoided when unnecessary. Please replace "d'" with "di".

File: it/mail/chrome/messenger/activity.properties [hunk 11]
```diff
@@ -75,9 +75,9 @@ renamedFolder=Rinominata cartella da #1 a #2
 indexing=Indicizzazione messaggi in corso
 # LOCALIZATION NOTE (indexingFolder): #1 is a folder name
 indexingFolder=Indicizzazione messaggi in #1
-indexingStatusVague=Individuazione dei messaggi da indicizzare
+indexingStatusVague=Determinazione dei messaggi da indicizzare
```
Comment (fenderman79) [inline it/mail/chrome/messenger/activity.properties]: CORRECTIONS NEEDED. 

"Individuazione" in the original text was just fine.

File: it/mail/chrome/messenger/activity.properties [hunk 12]
```diff
@@ -75,9 +75,9 @@ renamedFolder=Rinominata cartella da #1 a #2
...
 # LOCALIZATION NOTE (indexingFolderStatusVague): #1 is a folder name
-indexingFolderStatusVague=Individuazione dei messaggi da indicizzare in #1
+indexingFolderStatusVague=Determinazione dei messaggi da indicizzare in #1
```
Comment (fenderman79) [inline it/mail/chrome/messenger/activity.properties]: CORRECTIONS NEEDED. 

Same as #78

File: it/mail/chrome/messenger/addressbook/addressBook.properties [hunk 13]
```diff
@@ -57,7 +57,7 @@ editVCardTitleWithDisplayName=Modifica vCard per %S
 
 ## LOCALIZATION NOTE (cardRequiredDataMissingMessage):  do not localize \n
 cardRequiredDataMissingMessage=Devi inserire almeno uno fra questi campi:\nE-mail, Nome, Cognome, Nome visualizzato, Organizzazione.
-cardRequiredDataMissingTitle=Mancano informazioni obbligatorie
+cardRequiredDataMissingTitle=Non sono presenti informazioni obbligatorie
```
Comment (fenderman79) [inline it/mail/chrome/messenger/addressbook/addressBook.properties]: CORRECTIONS NEEDED. 

"Mancano" in the original text was just fine.

File: it/mail/chrome/messenger/addressbook/addressBook.properties [hunk 14]
```diff
@@ -146,7 +146,7 @@ failedToExportMessageFileAccessDenied=Esportazione rubrica non riuscita, accesso
 
 # For getting authDN for replication using dlg box
 AuthDlgTitle=Replica rubrica LDAP
-AuthDlgDesc=Inserire nome utente e password per accedere al server di directory.
+AuthDlgDesc=Per accedere al server di directory, inserire nome utente e password.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/addressbook/addressBook.properties]: CORRECTIONS NEEDED. 

The main clause should always be placed before the subordinate clause. The original text in red was just fine.

File: it/mail/chrome/messenger/addressbook/pref-directory-add.dtd [hunk 15]
```diff
@@ -51,7 +51,7 @@
 <!ENTITY advancedOptionsButton.label       "Opzioni avanzate">
 <!ENTITY advancedOptionsButton.accesskey   "A">
 <!ENTITY General.tab                       "Generale">
-<!ENTITY Offline.tab                       "Offline">
+<!ENTITY Offline.tab                       "offline">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/addressbook/pref-directory-add.dtd]: CORRECTIONS NEEDED. 

Probably "Offline" (with capital O) was correct.

File: it/mail/chrome/messenger/am-addressing.dtd [hunk 16]
```diff
@@ -1,54 +1,54 @@
...
 <!ENTITY autoQuote.accesskey                  "I">
 <!-- LOCALIZATION NOTE (then.label): This will concatenate with the 4 strings that follow. -->
-<!ENTITY then.label                           "dopodiché">
+<!ENTITY then.label                           "In seguito,">
 <!ENTITY then.accesskey                       "o">
-<!ENTITY aboveQuote.label                     "inizia la risposta sopra il testo a cui si risponde">
-<!ENTITY belowQuote.label                     "inizia la risposta sotto il testo a cui si risponde">
-<!ENTITY selectAndQuote.label                 "seleziona il testo a cui si risponde">
-<!ENTITY place.label                          "e metti la firma">
+<!ENTITY aboveQuote.label                     "inizia la mia risposta sopra il testo citato">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-addressing.dtd]: CORRECTIONS NEEDED. 

Here the possessive pronoun "mia" is redundant. Moreover, it's not used in the next string (#27), so I would avoid "mia" also to improve the overall consistency of the text.

File: it/mail/chrome/messenger/am-dsn.dtd [hunk 17]
```diff
@@ -1,16 +1,16 @@
 <!ENTITY pane.title                 "Notifica dello stato di consegna">
-<!ENTITY useGlobalPrefs.label       "Use my global Delivery Status Notification preferences for this account">
+<!ENTITY useGlobalPrefs.label       "Utilizza le mie preferenze globali per la Notifica dello stato di consegna per questo account">
 <!ENTITY useGlobalPrefs.accesskey   "U">
-<!ENTITY useCustomPrefs.label       "Customize Delivery Status Notification for this account">
+<!ENTITY useCustomPrefs.label       "Personalizza la Notifica dello stato di consegna per questo account">
 <!ENTITY useCustomPrefs.accesskey   "C">
-<!ENTITY requestAlways.label        "When sending messages, always request a Delivery Status Notification">
+<!ENTITY requestAlways.label        "Durante l'invio dei messaggi, richiedi sempre una Notifica dello stato di consegna">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-dsn.dtd]: CORRECTIONS NEEDED. 

The main clause should always be placed before the subordinate clause. The correct entry should be "Richiedi sempre una Notifica dello stato di consegna durante l'invio dei messaggi".

File: it/mail/chrome/messenger/am-mdn.dtd [hunk 18]
```diff
@@ -1,24 +1,24 @@
...
 <!ENTITY useGlobalPrefs.accesskey     "u">
-<!ENTITY useCustomPrefs.label         "Personalizza le opzioni per le ricevute di ritorno di questo account">
+<!ENTITY useCustomPrefs.label         "Personalizza le ricevute di ritorno per questo account">
 <!ENTITY useCustomPrefs.accesskey     "P">
-<!ENTITY requestReceipt.label         "Quando si spediscono i messaggi chiedi sempre la Ricevuta di ritorno">
+<!ENTITY requestReceipt.label         "Durante l'invio dei messaggi, richiedi sempre una ricevuta di ritorno">
 <!ENTITY requestReceipt.accesskey     "Q">
 <!ENTITY receiptArrive.label          "Quando arriva una ricevuta:">
-<!ENTITY leaveIt.label                "Lasciala nella posta in arrivo">
+<!ENTITY leaveIt.label                "Lasciala nella mia posta in arrivo">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-mdn.dtd]: CORRECTIONS NEEDED. 

The possessive pronoun "mia" is redundant. It's obviously _mine_ inbox, and not someone else's... that's why I would remove "mia".

File: it/mail/chrome/messenger/am-mdn.dtd [hunk 19]
```diff
@@ -1,24 +1,24 @@
...
 <!ENTITY leaveIt.accesskey            "L">
 <!-- LOCALIZATION NOTE moveToSent.label Translate: 'Sent' according to Netscape glossary -->
-<!ENTITY moveToSent.label             "Spostala nella cartella &quot;Posta inviata&quot;">
+<!ENTITY moveToSent.label             "Spostala nella mia cartella &quot;Posta inviata&quot;">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-mdn.dtd]: CORRECTIONS NEEDED. 

Same as #9 above.

File: it/mail/chrome/messenger/am-server-top.dtd [hunk 20]
```diff
@@ -66,15 +66,15 @@
...
 <!-- LOCALIZATION NOTE (maxMessagesEnd.label) : see note for maxMessagesStart.label -->
 <!ENTITY maxMessagesEnd.label "messaggi">
-<!ENTITY alwaysAuthenticate1.label "Always request authentication">
+<!ENTITY alwaysAuthenticate1.label "Richiedi sempre l'autenticazione">
 <!ENTITY alwaysAuthenticate1.accesskey "w">
-<!ENTITY abbreviate.label "Mostra nome del gruppo di discussione nel pannello della posta così:">
-<!ENTITY abbreviateOn.label "Nomi completi (Per esempio, 'netscape.public.mozilla.mail-news')">
-<!ENTITY abbreviateOff.label "Nomi abbreviati (Per esempio, 'n.p.m.mail-news')">
+<!ENTITY abbreviate.label "Mostra nome Newsgroup nel pannello E-mail come:">
+<!ENTITY abbreviateOn.label "Nomi completi (Ad esempio, 'netscape.public.mozilla.mail-news')">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-server-top.dtd]: CORRECTIONS NEEDED. 

The sentence isn't interrupted by a full stop - a bracket is used instead - therefore "Ad" must be replaced by "ad".

File: it/mail/chrome/messenger/am-server-top.dtd [hunk 21]
```diff
@@ -66,15 +66,15 @@
...
+<!ENTITY abbreviateOff.label "Nomi abbreviati (Ad esempio, 'n.p.m.mail-news')">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-server-top.dtd]: CORRECTIONS NEEDED. 

Same as above.

File: it/mail/chrome/messenger/am-smime.properties [hunk 22]
```diff
@@ -1,20 +1,20 @@
...
 
-encryption_needCertWantSame=È necessario specificare un certificato che gli altri possano usare quando inviano qui messaggi cifrati. Si vuole usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_wantSame=Usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_needCertWantToSelect=È necessario specificare un certificato che gli altri possano usare per rispondere con messaggi cifrati. Configurare un certificato di cifratura adesso?
-signing_needCertWantSame=Si deve specificare anche un certificato da usare per la firma digitale dei propri messaggi. Usare lo stesso certificato per firmare digitalmente i propri messaggi?
-signing_wantSame=Usare lo stesso certificato per firmare i propri messaggi?
-signing_needCertWantToSelect=Si deve specificare anche un certificato per la firma digitale dei propri messaggi. Configurare un certificato per firmare i messaggi digitalmente?
+encryption_needCertWantSame=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-smime.properties]: CORRECTIONS NEEDED. 

Elision and apocopation should be avoided when unnecessary. Please replace "t'" with "ti".

File: it/mail/chrome/messenger/am-smime.properties [hunk 23]
```diff
@@ -1,20 +1,20 @@
...
+encryption_wantSame=Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?
+encryption_needCertWantToSelect=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi configurare un certificato per la crittografia ora?
```
Comment (fenderman79) [inline it/mail/chrome/messenger/am-smime.properties]: CORRECTIONS NEEDED. 

Same as above.

File: it/mail/chrome/messenger/appleMailImportMsgs.properties [hunk 24]
```diff
@@ -54,11 +54,10 @@
 # Error Message
 ## @name APPLEMAILIMPORT_MAILBOX_BADPARAM
 ## @loc None
-2003=Si è verificato un errore interno, importazione non riuscita. Tentare nuovamente l'importazione.
+2003=Si è verificato un errore interno, importazione non riuscita. Riprova a importare di nuovo.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/appleMailImportMsgs.properties]: CORRECTIONS NEEDED. 

"Riprova" and "di nuovo" make this sentence redundand. Please replace "Riprova" with "Prova".

File: it/mail/chrome/messenger/comm4xMailImportMsgs.properties [hunk 25]
```diff
@@ -58,10 +58,10 @@
 # Error Message
 ## @name COMM4XMAILIMPORT_MAILBOX_BADPARAM
 ## @loc None
-2003=Si è verificato un errore interno, importazione non riuscita. Tentare nuovamente l'importazione.
+2003=Si è verificato un errore interno, importazione non riuscita. Riprova a importare.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/comm4xMailImportMsgs.properties]: CORRECTIONS NEEDED. 

In order to keep the consistency, please use the same sentence used in another file (that is, replace "Riprova a importare" with "Prova ad importare di nuovo")

File: it/mail/chrome/messenger/custom.properties [hunk 26]
```diff
@@ -1,2 +1,2 @@
-headerRemoved=You have removed header from the list. This may require resetting of the search filter criteria.
-colonInHeaderName=Uno dei filtri utilizza un intestazione personalizzata contenente caratteri non validi, quali ad esempio ':', caratteri non stampabili, caratteri non ASCII, caratteri nel campo di valori 128-255. Rimuovere i caratteri non validi e riprovare.
+headerRemoved=Hai rimosso l'intestazione dalla lista. Ciò potrebbe richiedere il ripristino dei criteri del filtro di ricerca.
+colonInHeaderName=L'intestazione che hai inserito contiene un carattere non valido, come ':', carattere non stampabile, carattere non ASCII o un carattere ASCII a 8 bit. Rimuovi il carattere non valido e riprova.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/custom.properties]: CORRECTIONS NEEDED. 

I would rephrase this sentence as follows:  +colonInHeaderName=L'intestazione che hai inserito contiene un carattere non valido (ad esempio ':'), un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Rimuovi il carattere non valido e riprova.

File: it/mail/chrome/messenger/filter.properties [hunk 27]
```diff
@@ -1,71 +1,71 @@
...
+invalidCustomHeader=Uno dei tuoi filtri utilizza un'intestazione personalizzata che contiene un carattere non valido, come ':', un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Modifica il file msgFilterRules.dat, che contiene i tuoi filtri, per rimuovere i caratteri non validi dalle intestazioni personalizzate.
+continueFilterExecution=L'applicazione del filtro %S non è riuscita. Vuoi continuare ad applicare i filtri?
+promptTitle=Esecuzione filtri
+promptMsg=Stai attualmente filtrando i messaggi.\nVuoi continuare ad applicare i filtri?
 stopButtonLabel=Interrompi
 continueButtonLabel=Continua
-cannotEnableFilter=Questo filtro è stato probabilmente creato da una versione successiva di Mozilla/Netscape. Non è possibile attivare questo filtro in quanto il formato non è supportato.
+cannotEnableFilter=Il filtro è stato probabilmente creato da una versione successiva di mozilla/netscape. Non puoi attivare questo filtro in quanto il formato non è supportato.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/filter.properties]: CORRECTIONS NEEDED. 

Mozilla and Netscape are registered trademarks, and need to be capitalized.

File: it/mail/chrome/messenger/filter.properties [hunk 28]
```diff
@@ -1,71 +1,71 @@
...
-logCopyStr=copiato l'id del messaggio = %1$S a %2$S
+logCopyStr=copiato id del messaggio = %1$S a %2$S
 # LOCALIZATION NOTE(filterLogDetectStr)
 # %1$S=filter name %2$S=author, %3$S=subject, %4$S=date
 filterLogDetectStr=Applicato filtro "%1$S" al messaggio da %2$S - %3$S a %4$S
-filterAction2=cambiata la priorità
+filterAction2=priorità modificata
 filterAction3=eliminato
-filterAction4=marcato come letto
+filterAction4=contrasseganto come letto
```
Comment (fenderman79) [inline it/mail/chrome/messenger/filter.properties]: CORRECTIONS NEEDED. 

There is a typo. Please replace "contrasseganto" with "contrassegnato".

File: it/mail/chrome/messenger/imapMsgs.properties [hunk 29]
```diff
@@ -365,107 +365,107 @@
...
 ## @loc None
 # LOCALIZATION NOTE (5115): %S is the server hostname
-5115=The Kerberos/GSSAPI ticket was not accepted by the IMAP server %S. Please check that you are logged in to the Kerberos/GSSAPI realm.
+5115=Il ticket Kerberos/GSSAPI non è stato accettato dal server IMAP %S. Controlla di aver effettuato l'accesso al reame Kerberos/GSSAPI.
 
 ## @name IMAP_LOGIN_DISABLED
 ## @loc None
 # LOCALIZATION NOTE (5116): %S is the account name
-5116=You cannot log in to %S because the server doesn't allow plaintext authentication without STARTTLS or SSL/TLS. Try enabling connection security or secure authentication in the account settings.
+5116=Non puoi accedere a %S perché il server non consente autenticazione con testo semplice senza STARTTLS o SSL/TLS. Prova ad attivare la sicurezza connessione o l'autenticazione sicura nelle impostazioni account.
```
Comment (fenderman79) [inline it/mail/chrome/messenger/imapMsgs.properties]: CORRECTIONS NEEDED. 

An article is missing. Please add "l'" before "autenticazione con testo semplice".

File: it/mail/chrome/messenger/junkMailInfo.dtd [hunk 30]
```diff
@@ -1,7 +1,7 @@
-<!ENTITY window.title  "Informazioni sui messaggi di posta indesiderata">
+<!ENTITY window.title  "Informazioni sulle Posta indesiderata">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/junkMailInfo.dtd]: CORRECTIONS NEEDED. 

There is a typo. Please replace "sulle" with "sulla".

File: it/mail/chrome/messenger/accountCreation.properties (hunk 10 above)
Comment (shermandickman) [inline it/mail/chrome/messenger/accountCreation.properties]: @fenderman79  Do you mean:

diinserire or di inserire?

File: it/mail/chrome/messenger/am-smime.properties (hunk 22 above)
Comment (shermandickman) [inline it/mail/chrome/messenger/am-smime.properties]: @fenderman79 again ti inviano or tiinviano?

File: it/mail/chrome/messenger/accountCreation.properties (hunk 10 above)
Comment (fenderman79) [inline it/mail/chrome/messenger/accountCreation.properties]: Hi Sherman, 
The correct entry is "di inserire". 
Best, 
Andrea
Via InoMail on Android-------- Messaggio originale --------24 agosto 2016 03:52:52 CEST, Sherman Dickman <notifications@github.com> ha scritto:
In it/mail/chrome/messenger/accountCreation.properties:
> @@ -41,7 +41,7 @@ config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
>  config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
>  incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
>  outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
> -please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.
> +please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.

@fenderman79  Do you mean:

diinserire or di inserire?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/postbox/localizations","title":"postbox/localizations","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/postbox/localizations"}},"updates":{"snippets":[{"icon":"PERSON","message":"@shermandickman in #26: @fenderman79  Do you mean:\r\n\r\ndiinserire or di inserire?"}],"action":{"name":"View Pull Request","url":"https://github.com/postbox/localizations/pull/26/files/63340a85f6ae3d0662a2998ad4efa42b76afe2a3#r75983453"}}}

File: it/mail/chrome/messenger/am-smime.properties (hunk 22 above)
Comment (fenderman79) [inline it/mail/chrome/messenger/am-smime.properties]: Like the previous one, the correct entry here is "ti inviano". 
Best, 
Andrea 
Via InoMail on Android-------- Messaggio originale --------24 agosto 2016 03:58:28 CEST, Sherman Dickman <notifications@github.com> ha scritto:
In it/mail/chrome/messenger/am-smime.properties:
>  
> -encryption_needCertWantSame=È necessario specificare un certificato che gli altri possano usare quando inviano qui messaggi cifrati. Si vuole usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
> -encryption_wantSame=Usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
> -encryption_needCertWantToSelect=È necessario specificare un certificato che gli altri possano usare per rispondere con messaggi cifrati. Configurare un certificato di cifratura adesso?
> -signing_needCertWantSame=Si deve specificare anche un certificato da usare per la firma digitale dei propri messaggi. Usare lo stesso certificato per firmare digitalmente i propri messaggi?
> -signing_wantSame=Usare lo stesso certificato per firmare i propri messaggi?
> -signing_needCertWantToSelect=Si deve specificare anche un certificato per la firma digitale dei propri messaggi. Configurare un certificato per firmare i messaggi digitalmente?
> +encryption_needCertWantSame=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?

@fenderman79 again ti inviano or tiinviano?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/postbox/localizations","title":"postbox/localizations","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/postbox/localizations"}},"updates":{"snippets":[{"icon":"PERSON","message":"@shermandickman in #26: @fenderman79 again ti inviano or tiinviano?"}],"action":{"name":"View Pull Request","url":"https://github.com/postbox/localizations/pull/26/files/63340a85f6ae3d0662a2998ad4efa42b76afe2a3#r75983874"}}}
